src/Codeception/Module/Db.php

```diff
@@ -272,8 +272,25 @@ private function readSql()
 
     private function connect()
     {
+        $options = [];
+ 
+        if( array_key_exists('ssl_key', $this->config)
+            AND !empty($this->config['ssl_key'])) {
```

Comment:
`AND` ?? No one writes like that 😊
Please follow PSR and replace it with `&&`.

Reply:
Also if statement should be formatted as `if (array_key_exists`. (space before opening parenthesis).